framework/plugins/depot/depot.go

```
@@ -36,24 +37,37 @@ func (depot *Depot) GroupTaskBimap() bimap.Bimap[string, string] {
 	return depot.groupTaskBimap
 }
 
-func (depot *Depot) Write(ctx context.Context, id string) error {
+func (depot *Depot) Write(ctx context.Context, ids ...string) error {
 	data := kernel.GetPluginData(ctx).GetWithDefault("dependencies", &PluginData{}).(*PluginData)
+	// what to do if there are multiple ids? (i have only one dependency list)
+	if len(ids) != 1 {
+		return fmt.Errorf("i do not know what to do with multiple task ids")
```

Please return an error reflecting that multitask scheduling is not supported